cmd/upgrade_test.go

```diff
@@ -0,0 +1,56 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/exercism/cli/cli"
+	"github.com/stretchr/testify/assert"
+)
+
+type fakeCLI struct {
+	*cli.CLI
+}
```

Comment:
How does that saying go? 

> "If you wake up in the mornin'and you can't think of anything but singin' first...then you're supposed to be a singer"  ❤️ 

Yeah, I agree 100%. It will simplify things and make the test clearer. FYI, we don't have an actual test for cli.Upgrade "yet", but that will be a separate PR. 

>Should this be 2.0.0 for the sake of clarity, since the endpoint returns 2.0.0.?

 I saw your comment above and thought maybe we could be more explicit in the test seeing as I am just using that the version as a check that Upgrade is not being called for updated clients. So I'm glad you slept on this for a bit and came back with the suggestion. I will update and push back for final review 👍